### PR TITLE
Feature/ngx charts

### PR DESCRIPTION
**Pull request by richavyas**
## Description

NGX Charts Research repo

### What's included?

- Integration with ngx-charts latest version
- Bar Chart for Listener's Records Size Data for an hour
- Listener's volume api's data has been used

What's coming next:
- Line Chart for Router Latency
- Area Chart for Records

#### Test Steps

- [ ] rm -rf node_modules dist deploy tmp
- [ ] npm i
- [ ] ng serve

ngx-charts-records-size

### CODE REVIEW

**Comment by richavyas** [comment]
- Area Chart added 
- Line Chart added

swimlane-charts

File: src/app/components/components/ngx-charts/data.ts [hunk 1]
```diff
@@ -0,0 +1,1894 @@
+export let sourcesData: any = [{
```
Comment (emoralesb05) [inline src/app/components/components/ngx-charts/data.ts]: I think we should have a smaller size (maybe 20 items) file locally and try to hit a `hiroku` endpoint to load data from the `mock-data` later on..

**Comment by kyleledbetter** [comment]
Made a lot of updates:
ngx-charts

**Comment by richavyas** [comment]
LGTM!!!

**Pull request merged**